website_sale_order_type/views/website_config_settings_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="view_website_config_settings" model="ir.ui.view">
+        <field name="name">res.config.settings.inherit</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website_sale.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='sale_shipping_settings']" possition="inside">
+                <div class="col-xs-12 col-md-6 o_setting_box" id="sale_order_type_setting">
```

Comment:
Hi @ValentinoDefelice thanks for this PR
I would put this part in the "Sales" section 
https://github.com/odoo/odoo/blob/11.0/addons/website_sale/views/res_config_settings_views.xml#L333